nestkernel/nest_types.h

```diff
@@ -32,6 +32,19 @@
 // Generated includes:
 #include "config.h"
 
+#ifdef _OPENMP
```

Comment:
- Please add a comment on the purpose of these macros, including an example for their use.
- I think it is strange that these macros are defined in `nest_types.h`, since they have nothing to do with types. Wouldn't `vp_manager.h` be a more logical placement?